ja.lproj/Localizable.strings

```diff
@@ -1,117 +1,118 @@
 //
-// Translation by lisiyaki
+// Translation by KAGUYA-1137
```

Comment:
Please keep all contributor names

Reply:
Added my name below but it failed the check:(